### ISSUE TITLE
After updating to 15.78 Smarttubenext

### ISSUE THREAD

**Issue by keoghrogers7788** [yuliskov/SmartTubeNext#1769]
I know ur probably seen a ton of these, but after installing the 15.78 SmartTubeNext update. The one thing I haven't seen mentioned yet is that it will play videos that are live, but all the others. It acts like it will start then crashes back to the top of the list of. Whatever page ur in the goes blank again and when it comes back on its on the video you were just trying to watch. It's doing this on my android phone and all of my fire tv devices!! Before I had downloaded the update the videos were working, but the were buffering. Like every 3 seconds!!
One other thing that was happening the past couple days to me atleast. Was while watching SmartTubeNext I kept having one specific app that kept loading all on its own. That app was the Peacock app on my amazon fire tv. I do wonder though if any of these issues could be because of Amazon's latest update that the have tinkered with the Developer settings?

**Comment by shumway77** [comment]
Not tested with live streams, but I also cannot play any video after the 15.78 update just as described above.

**Comment by nesterenkovy** [comment]
After installing the update, the video stopped playing. The application crashes with an error - it is impossible to show the video. (tv box android6.0).
eset the default settings - everything works. However, as soon as I logged into my account, the problem returned.
I rolled back to the previous version - everything works stably.
A similar situation on two tv-boxes.

**Comment by Cebihub** [comment]
Yeah just updated and its broke and I tried all the normal methods and even did the whole reinstall and downloading from the site again.
Like people are saying, something in this update broke playing videos.

The beta build seems fine though, used that and had no issues.

**Comment by UZOgithub** [comment]
With the new update (15.78 smarttube stable) I can't start any video on my FireTVstick (Tank)!!!

Every time I start a video the mainscreen appears.

edit: It seems the problem is only when I am logged in with my youtube account. Without account it works.

**Comment by shopkeeper1970** [comment]
> With the new update (15.78 smarttube stable) I can't start any video on my FireTVstick (Tank)!!!
> 
> Every time I start a video the mainscreen appears.
> 
> It seems the problem is only when I am logged in with my youtube account. Without account it works.
> 
> edit: only livestreams work when logged in.

Same problem after update. hope its fixed soon.

**Comment by danheinz** [comment]
After the update, I am having the same issue as all the ones above on my firestick.

**Comment by evoactivity** [comment]
@reactii0n you joined github just to be a dick. You're using software developed by 1 person released for free and open source. If you're not a dumbass why not provide the fix yourself.

**Comment by robprid** [comment]
After update last night I have been having terrible buffering issues (every three seconds) . Thanks again for a wonderful app , makes watching Youtube a pleasure

**Comment by ashutosh44ks** [comment]
Same issue.
First buffering every 3 seconds then I updated then it no plays everything

**Comment by shumway77** [comment]
After installing the 15.79 beta, everything is fine again.

**Comment by rksleung** [comment]
The issue with the latest stable version is that all video except live stream crash. And I noticed that it only crashes when being signed in. When I sign out, I can play all video fine.

**Comment by UZOgithub** [comment]
With the 15.81 (stable) update the problems are fixed. Many thanks!!!

**Comment by Chealer** [comment]
@keoghrogers7788 : Does this persist with any current version? If so, please provide reproduction steps.
